playbook/app/pb_kits/playbook/pb_date_picker/date_picker_helper.ts

```diff
@@ -54,7 +73,7 @@ const datePickerHelper = (config, scrollContainer) => {
     }
   }
   const calendarResizer = () => {
-    const cal = document.querySelector(`#cal-${pickerId}.open`)
+    const cal = document.querySelector(`#cal-${pickerId}.open`) as HTMLElement | null
     const parentInput = cal.parentElement
     if (cal.getBoundingClientRect().right > window.innerWidth) {
```

Comment:
... use the safe operator in this conditional

```suggestion
    if (cal?.getBoundingClientRect().right > window.innerWidth) {
```